src/EventSubscriber/SearchNoHitEventSubscriber.php

```diff
@@ -57,17 +64,101 @@ public static function getSubscribedEvents(): array
      * Event handler.
      *
```

Comment:
It would have been nice with a little description about what we are doing, utilises cache to prevent not need processing etc.

Reply:
Fixed. But git blame that line please ;-)